### DISCUSSION TITLE
Old Solo Plots not showing on 1.2.0 GUI

### DISCUSSION THREAD

**Discussion by tksalarzai** [Chia-Network/chia-blockchain#7233] [Support]
i downloaded the 1.2.0 NEW VERSION and my old plots (solo plots) are now not showing on GUI ??? i removed directories from gui and re added all plots but still not appearing

**Comment by ZielReaver** [comment]
give it a few mins then hit refresh plots in the plot section same thing happening to a lot of people.

**Comment by brianwfreeman** [comment]
The refresh trick is not working for me on MacOS. No plots showing up.  Ugh.

**Reply by ZielReaver** [reply]
if refresh is not working only thing I can think if is what I did before I found out about the auto refresh not work. I went and uninstall chia then went and removed the .chia folder in user then went into appdata under local and roaming and removed the chia blockchain folders. Finally went and reinstalled the beta gui 1.1.795 and it started showing my plots again but had to rebuild the node and sync wallet again.

You can try just uninstalling the 1.2.0 and reinstalling the beta 1.1.795 and see if that works if not uninstall and delete those folders and try installing beta gui again.

**Comment by brianwfreeman** [comment]
I finally figured it out today.  It was a config.yaml setting that made the
GUI act like a harvester only.




Brian W. Freeman

Attorney at Law

Certified Specialist in CA Workers' Compensation Law



http://WorkersCompensationLawyerInc.com
<http://workerscompensationlawyerinc.com/>

Phone & Fax: (844) 4 AVG JOE

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
